tests/ModelTest.php

```diff
@@ -81,12 +87,24 @@ public function testQualifyColumn(): void
         $this->assertEquals('users.name', $sqlUser->qualifyColumn('name'));
     }
 
+    private function makeUser(array $overrides = []): User
```

Comment:
The `$overrides` is never used. I don't think we need it.

Reply:
@GromNaN 

Thanks for replying !!

i fixed [it](https://github.com/mongodb/laravel-mongodb/pull/3416/commits/6ecd5279422805597c2caa1713672ca4bf35f133) ! 🙌